src/components/NewDesign/Swap/Swap.tsx

```diff
@@ -0,0 +1,424 @@
+          </Typography>
+        </Box>
+        <ExchangeAmountInput
+          value={amountFrom}
+          className={classes.amountInput}
+          style={{ transform: swap !== null ? swap ? 'translateY(104px)' : 'translateY(0px)' : '' }}
+          setValue={value => {
+            if (value.match(/^\d*\.?\d*$/)) {
+              setAmountFrom(value)
+              updateEstimatedAmount((+value * tax).toString())
+            }
+          }}
+          placeholder={'0.0'}
+          onMaxClick={() => {
+            if (tokenToIndex !== null && tokenFromIndex !== null) {
+              if (swap) {
+                setAmountTo(printBN(tokens[tokenToIndex].balance, tokens[tokenToIndex].decimal))
+                updateFromEstimatedAmount(printBN(tokens[tokenToIndex].balance, tokens[tokenToIndex].decimal))
+              } else {
+                setAmountFrom(printBN(tokens[tokenFromIndex].balance, tokens[tokenFromIndex].decimal))
+                updateEstimatedAmount(printBN(tokens[tokenFromIndex].balance, tokens[tokenFromIndex].decimal))
+              }
+            }
+          }}
+          tokens={swap ? tokenY.map((
+            {
+              symbol,
+              name,
+              logoURI
+            }) => ({
+            symbol,
+            name,
+            logoURI
+          }))
+            : tokens.map((
+              {
+                symbol,
+                name,
+                logoURI
+              }) => ({
+              symbol,
+              name,
+              logoURI
+            }))}
+          current={
+            tokenFromIndex !== null
+              ? tokens[tokenFromIndex].symbol
+              : null}
+          onSelect={(name: string) => {
+            console.log(name)
+            setTokenFromIndex(
+              tokens.findIndex((token) => {
+                return name === token.symbol
+              })
+            )
+          }}
+        />
+        <Box className={classes.tokenComponentTextContainer}>
+          <Box className={classes.swapArrowBox}>
+            <CardMedia image={SwapArrows}
+              style={
+                {
+                  transform: swap !== null
+                    ? swap
+                      ? 'rotate(180deg)'
+                      : 'rotate(0deg)'
+                    : ''
+                }
+              }
+              className={classes.swapArrows} onClick={() => {
+                if (tokenToIndex !== null) {
+                  swap !== null
+                    ? setSwap(!swap)
+                    : setSwap(true)
+                }
+              }} />
+          </Box>
+          <Typography className={classes.tokenComponentText}>To (Estd.)</Typography>
+          <Typography className={classes.tokenComponentText}>
+          Balance: {tokenToIndex !== null
+              ? swap
+                ? printBN(tokens[tokenFromIndex ?? 0].balance, tokens[tokenFromIndex ?? 0].decimal)
```

Comment:
why tokenFrom? just use 0

Reply:
same thing like above, i tried to change position of balance depends on swap